component/common/pom.xml

```diff
@@ -96,6 +99,7 @@
                             javax.crypto.spec,
                             com.fasterxml.jackson.core,
                             com.fasterxml.jackson.databind,
+                            org.apache.log4j,
```

Comment:
Would this be needed now since we are using slf4j API?

Reply:
yes, you are correct. Let's remove this.